server/plugin/webhook.go

```diff
@@ -342,13 +343,18 @@ func (p *Plugin) handlePRDescriptionMentionNotification(event *github.PullReques
 
 func (p *Plugin) postIssueEvent(event *github.IssuesEvent) {
 	repo := event.GetRepo()
+	issue := event.GetIssue()
+	action := event.GetAction()
+
+	if time.Now().Unix()-issue.GetCreatedAt().Unix() <= 3 && action == "labeled" {
```

Comment:
Instead of relying on the hard to remember semantic of `Unix()` should [`Sub`](https://pkg.go.dev/time#Time.Sub) get used instead?